docs/decisions/0005-optional-import-of-confluent-kafka.rst

```diff
@@ -0,0 +1,60 @@
+4. Optional import of confluent-kafka
+#####################################
+
+Status
+******
+
+Accepted
+
+Context
+*******
+
+* `confluent-kafka`_ is a library written and maintained by Confluent, our managed instance provider (see :doc:`0004-kafka-managed-hosting`). The library abstracts out the work for sending and receiving events to and from the Kafka cluster and converting them into message objects.
+* confluent-kafka in turn is a wrapper around a C library called `librdkafka`_ (distributed as `librdkafka_dev`)
+* librdkafka-dev does not currently have a compiled binary available for Linux/aarch64
```

Comment:
Probably should mention here that this causes a problem for people trying to run Tutor Linux images on M1 Macs.

Reply:
I'm going to argue on this one. I think that's too specific for this particular ADR. I don't want people to have to know what tutor is to understand it. If we want to emphasize why this is a problem, I would go for `librdkafka-dev does not currently have a compiled binary available for Linux/aarch64, which is a fairly common architecture for Open edX developers`

Reply:
I think it's important to include as background context, particularly because it's non-obvious that Linux/aarch64 isn't the _host_ OS.